packages/kbn-securitysolution-es-utils/src/delete_all_index/index.ts

```diff
@@ -22,9 +23,14 @@ export const deleteAllIndex = async (
 
     // resolve pattern to concrete index names
     const { body: resp } = await esClient.indices.getAlias(
-      {
-        index: pattern,
-      },
+      specifyAlias
+        ? {
+            name: pattern,
+            index: `${pattern}-*`,
+          }
+        : {
+            index: pattern,
+          },
```

Comment:
@marshallmain this is used by the lists functional tests to delete the lists index so making the change we discussed broke those tests, so I added a flag